buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/AndroidBinaryRule.groovy

```diff
@@ -0,0 +1,42 @@
+package com.uber.okbuck.experimental.bazel
+
+import com.uber.okbuck.rule.BuckRule
+
+final class AndroidBinaryRule extends BuckRule {
```

Comment:
Can we name all these classes with a `Bazel` prefix? This is to avoid confusing with the buck classes